### ISSUE TITLE
Document that view_init() arguments should be in degrees

### ISSUE THREAD

**Issue by dstansby** [matplotlib/matplotlib#14372]
The `view_init()` method for 3D axes should specify that the elevation and azimuth arguments are in degrees (as opposed to radians). The docstring to change is here: https://matplotlib.org/devdocs/api/_as_gen/mpl_toolkits.mplot3d.axes3d.Axes3D.html?highlight=view_init#mpl_toolkits.mplot3d.axes3d.Axes3D.view_init

**Comment by abhinavsagar** [comment]
Could I work on this ?

**Comment by RoryIAngus** [comment]
I have done this and added it to my fork found https://github.com/RoryIAngus/matplotlib

**Comment by RoryIAngus** [comment]
Can you please advise what branch I should add this to and I will create a pull request.

Thanks

**Comment by timhoffm** [comment]
Generally, all pull requests should be made against master. We then backport to other branches if appropriate.

**Comment by RoryIAngus** [comment]
This line confused me, is it meant to mean a master branch on my copy of the forked repo? 
```
- Do not create the PR out of master, but out of a separate branch.
```

**Comment by timhoffm** [comment]
Yes.

**Comment by timhoffm** [comment]
Closed by #14438.
